src/components/markdown-renderer/replace-components/markmap/markmap-frame.tsx

```diff
@@ -12,14 +12,24 @@ export const MarkmapFrame: React.FC<MarkmapFrameProps> = ({ code }) => {
       return
     }
     const actualContainer = diagramContainer.current
-    Promise.all([import(/* webpackChunkName: "markmap" */ 'markmap-lib/dist/transform'), import(/* webpackChunkName: "markmap" */ 'markmap-lib/dist/view')])
-      .then(([transform, view]) => {
+    Promise.all([import(/* webpackChunkName: "markmap" */ 'markmap-lib/dist/transform'), import(/* webpackChunkName: "markmap" */ 'markmap-lib/dist/view'), import(/* webpackChunkName: "markmap" */ 'markmap-lib/dist/util/loader')])
+      .then(([transform, view, loader]) => {
         const svg: SVGSVGElement = document.createElementNS('http://www.w3.org/2000/svg', 'svg')
         svg.setAttribute('width', '100%')
         actualContainer.querySelectorAll('svg').forEach(child => child.remove())
         actualContainer.appendChild(svg)
-        const data = transform.transform(code)
-        view.Markmap.create(svg, {}, data)
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+        const { root, features } = transform.transform(code)
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-call,@typescript-eslint/no-unsafe-assignment
+        const { styles, scripts } = transform.getUsedAssets(features)
+        if (styles) {
+          loader.loadCSS(styles)
+        }
+        if (scripts) {
+          loader.loadJS(scripts, { getMarkmap: () => view.Markmap })
+            .catch(err => console.error(err))
+        }
+        view.Markmap.create(svg, {}, root)
```

Comment:
Do we really have to do this? D: and why? And where is it loaded from?

Reply:
The old syntax does not work anymore with the update, so yes we need to do this.
Afaik the used css and js is fetched from everything the lib provides and added to the display, so in theory we should get smaller svgs

Reply:
So it won't be fetched from any external server?

Reply:
as far as I know it won't